pallets/registrar/src/tests.rs

```diff
@@ -1017,10 +1098,20 @@ fn deregister_2_container_chains_in_same_block() {
         assert_eq!(
             Mock::mock().called_hooks,
             vec![
+                HookCall::InnerRegister(42.into()),
+                HookCall::InnerRegister(43.into()),
                 HookCall::MarkedValid(42.into()),
+                HookCall::InnerScheduleParaUpgrade(42.into()),
                 HookCall::MarkedValid(43.into()),
+                HookCall::InnerScheduleParaUpgrade(43.into()),
+                HookCall::InnerScheduleParaDowngrade(42.into()),
+                HookCall::InnerScheduleParaDowngrade(43.into()),
                 HookCall::Deregistered(42.into()),
                 HookCall::Deregistered(43.into()),
+                HookCall::InnerDeregisterWeight,
+                HookCall::InnerDeregisterWeight,
+                HookCall::InnerDeregister(42.into()),
+                HookCall::InnerDeregister(43.into()),
```

Comment:
This is nice, it shows the entire parachain cycle and which hooks are executed in which order.